lib/server.js

```diff
@@ -2,12 +2,12 @@ import log from './logger';
 import { server as baseServer, routeConfiguringFunction } from 'appium-base-driver';
 import AndroidDriver from './driver';
 
-
 async function startServer (port, host) {
   let d = new AndroidDriver();
   let router = routeConfiguringFunction(d);
   let server = baseServer(router, port, host);
   log.info(`AndroidDriver server listening on http://${host}:${port}`);
+  d.server = server;
```

Comment:
This will work for the running the `appium-android-driver` tests, but this server does not exist for Appium itself, where the server is created here: https://github.com/appium/appium/blob/master/lib/main.js#L124

I'm not sure how this is intended to work.

Reply:
:( what would be the best way to get this server instance in the driver then?

Reply:
https://github.com/appium/appium/pull/10400